cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -544,6 +572,13 @@ mkConsensusTracers trSel verb tr nodeKern fStats = do
        <*> counting (liftCounting staticMeta name "slot-is-immutable" tr)
        <*> counting (liftCounting staticMeta name "node-is-leader" tr)
 
+   traceServedCount :: Maybe EKGDirect -> STM.TVar Int -> TraceChainSyncServerEvent blk -> IO ()
+   traceServedCount Nothing _ _ = pure ()
+   traceServedCount (Just ekgDirect) tHeadersServed ev =
+     when (isRollForward ev) $ do
+       count <- STM.modifyReadTVarIO tHeadersServed (+1)
```

Comment:
Are we still using TVars?  I had the impression that we were going to replace the use of TVars.

Reply:
Mostly due to this comment https://input-output-rnd.slack.com/archives/GR599HMFX/p1613382544290700

Reply:
@newhoggy, can you tell a bit more -- what should we be using instead?  I'm afraid I'm not up-to-date..

Reply:
Found the comment https://github.com/input-output-hk/cardano-node/pull/2366#discussion_r576062283

Reply:
I added @coot to seek clarification on this question.

Reply:
Good point, we should be using https://hackage.haskell.org/package/ekg-core-0.1.1.7/docs/System-Metrics-Counter.html